wagtailmetadata/templates/wagtailmetadata/parts/tags.html

```diff
@@ -20,13 +20,4 @@
 {% endif %}
 {% endblock opengraph %}
 
-{% block meta %}
-<meta itemprop="url" content="{{ object.get_meta_url }}">
```

Comment:
instead of removing these tags can you update them to use the `name` property instead of `itemprop`

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta